modules/jupyter-widget/package.json

```diff
@@ -36,14 +36,20 @@
     "@deck.gl/json": "8.7.11",
     "@deck.gl/layers": "8.7.11",
     "@deck.gl/mesh-layers": "8.7.11",
-    "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3",
+    "@jupyter-widgets/base": "^2 || ^3 || ^4",
```

Comment:
Is there a reason to remove `^1`? Or, is it known which version of jupyter notebook started to move to `^2`? Is this suggested by one of the current dev references?

Reply:
The [webpage](https://ipywidgets.readthedocs.io/en/stable/changelog.html#updates-for-widget-maintainers) says

> Update the @jupyter-widgets/base dependency version to include ^4 to work in JupyterLab 3.0. For example, if you had a dependency on @jupyter-widgets/base version ^2 || ^3, update to ^2 || ^3 || ^4 for your widget to work in classic Jupyter Notebook, JupyterLab 1, JupyterLab 2, and JupyterLab 3. See [#2472](https://github.com/jupyter-widgets/ipywidgets/pull/2472) for background.

However, it seems like keeping ^1.1.10 would be fine.

Reply:
I'd prefer to keep `^1.1.10` as well unless there's a conflicgt.